salt/modules/virt.py

```diff
@@ -250,7 +250,7 @@ def _gen_xml(name,
         # TODO: make bus and model parameterized, this works for 64-bit Linux
         context['controller_model'] = 'lsilogic'
 
-    if kwargs.get('enable_vnc', True):
+    if 'enable_vnc' in kwargs:
```

Comment:
And so `enable_vnc=False` supposed to be translated into `enable_vnc=True`? It should be as simple as this (assuming default is no VNC):
```python
context['enable_vnc'] = kwargs.get('enable_vnc') or False
```